app/configs/i18next.config.client.js

```diff
@@ -0,0 +1,28 @@
+import i18next from 'i18next';
```

Comment:
I'd rather this file be called `i18next.config.renderer.js`

Reply:
fixed